src/components/organisms/DataResultItems.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useMemo } from 'react';
+import { InlineAlert, Pane } from 'evergreen-ui';
+import DataResultItem from '../molecules/DataResultItem';
+import { useQuery } from '@apollo/client';
+import { METADATA, AllMetadataResult } from '../../queries/metadata';
+import Loading from '../atoms/Loading';
+import ErrorMessage from '../atoms/ErrorMessage';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../redux';
+
+const DataResultItems: React.FC = () => {
+    const { data, loading, error } = useQuery<AllMetadataResult>(METADATA);
+    const activeFilters = useSelector((state: RootState) => state.filter.filters);
```

Comment:
Since the filters are entirely limited to one page, I don't know if Redux is strictly necessary, or if you could just create a simple reducer with `useReducer` in this component.

That way you would limit the state to the relevant component.

Reply:
Since you're also just dealing with a list of tags, it might even be solvable with only a `useState`.

When updating a `useState` hook, you can write something like this;

```ts
setState((previousState) => ([...previousState, newTag]))
```

To add a new entry to the list.

Reply:
Though a reducer might open for more future functionality. 

Another benefit of using a `useReducer` here instead of redux is that you eliminate a dependency from the component subtree (i.e. you no longer require a provider or library).